binary/expression.proto

```diff
@@ -68,6 +77,18 @@ message Expression {
             repeated KeyValue key_values = 1;
         }
 
+        message Decimal {
+            // little-endian twos-complement integer representation of complete value (ignoring precision)
+            bytes value = 1;
+            
+            // declared scale of decimal literal
+            int32 scale = 2;
+            
+            // place to put decimal within twos-complement value.
+            int32 precision = 3;
+        
```

Comment:
Kill this extra newline. I'm going to open up a PR to wrangle all these random newlines everywhere and enforce a consistent style. It's starting to get a bit out of hand.

Reply:
Don't be a newline hater!